product_multi_company/migrations/13.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,24 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from openerp import SUPERUSER_ID, api
```

Comment:
```suggestion
from odoo import SUPERUSER_ID, api
```

Reply:
Isn't this normally handled via pre-commit? Or maybe it was a MQT script that checked it.